umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/configuration.md

```diff
@@ -60,3 +60,30 @@ Some of the backoffice users might be restricted to a specific part of the conte
 Images added through the RTE is by default added to the root of the Media library.
 
 Sometimes you might want to add the images to a specific folder. This folder can be configured using the "Image Upload Folder" setting.
+
+# Advanced Configuration
+
+The RTE can also accept advanced configuration through the `appSettings.json` file fed into the TinyMCE configuration. This is within `Umbraco` / `CMS` / `RichTextEditor` and allows you to configure the following:
```

Comment:
Updated the formatting for referencing the specific config section. This is how we do it other places in the docs, so though it would make sense to keep it consistent 💪 

```suggestion
The RTE can also accept advanced configuration through the `appSettings.json` file fed into the TinyMCE configuration. This is within `Umbraco:CMS:RichTextEditor` and allows you to configure the following:
```